src/Plugin.Maui.Audio/IAudioPlayer.cs

```diff
@@ -30,6 +30,11 @@ public interface IAudioPlayer : IDisposable
 	///</Summary>
 	double Balance { get; set; }
 
+	///<Summary>
+	/// Gets or sets the playback speed where 1 is normal speed.
```

Comment:
Are there minimum and maximum values on specific platforms?

Reply:
Yes, as I found on Android, it seems like it. And with AVAudioPlayer on iOS, on values 0.5 - 2.0 are supported. So the idea with the SpeedThresholdValueExceededException, maybe is not such a bad idea!?
[SpeedThresholdValueExceededException ](url)